server/app.js

```diff
@@ -88,13 +89,16 @@ const mongodbPromise = MongoClient.connect(mongoURI, {
   ssl: false,
   useNewUrlParser: true,
 }).then(
-  function (res) {
+  async function (res) {
     app.locals.connection = res
     mongodb = res.db()
     app.locals.appDb = res.db()
     app.locals.dataDb = res.db('dpdata')
     res.db().collection('sessions').drop()
 
+    if (await UserModel.hasNoAdmin(app.locals.appDb)) {
+      await UserModel.createFirstAdmin(app.locals.appDb, app.locals.dataDb)
+    }
```

Comment:
what do you think about combining `hasNoAdmin` in `createFirstAdmin`, so that we can never create another first admin? That would also make this just the 1 line `await UserModel.createFirstAdmin(app.locals.appDb, app.locals.dataDb)`

Reply:
I like it!